lib/stripe/instrumentation.rb

```diff
@@ -41,14 +44,17 @@ class RequestEndEvent
       attr_reader :user_data
 
       def initialize(duration:, http_status:, method:, num_retries:, path:,
-                     request_id:, user_data: nil)
+                     request_id:, user_data: nil, response: nil, request_header: nil, request_body: nil)
         @duration = duration
         @http_status = http_status
         @method = method
         @num_retries = num_retries
         @path = path
         @request_id = request_id
         @user_data = user_data
+        @response = response
+        @request_header = request_header
+        @request_body = request_body
```

Comment:
Instead of exposing the entire `response` object, can we make this expose only `response_header` and `response_body` to have it match request? We're a little reluctant to make `StripeResponse` part of the public API.